components/esp_modem/examples/modem_console/main/my_module_dce.cpp

```diff
@@ -0,0 +1,95 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Unlicense OR CC0-1.0
+ */
+/* Modem console example: Custom DCE
+
+   This example code is in the Public Domain (or CC0 licensed, at your option.)
+
+   Unless required by applicable law or agreed to in writing, this
+   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied.
+*/
+
+#include <cstring>
+#include "cxx_include/esp_modem_api.hpp"
+#include "cxx_include/esp_modem_dce_module.hpp"
+#include "generate/esp_modem_command_declare.inc"
+#include "my_module_dce.hpp"
+
+using namespace esp_modem;
+
+//
+// Define preprocessor's forwarding to dce_commands definitions
```

Comment:
Not sure if this is a leftover comment or a short description for the following sections.

Reply:
Not a leftover, it's a valid comment. Any suggestion for rewording? (this comment just introduces a block of macros that help with forwarding, since the standard `perfect forwarding` cannot handle certain scenarios (so we use these macros to make them perfect `perfect forwarding`)